core.js

```diff
@@ -2152,28 +2152,6 @@ class HTMLScriptElement extends HTMLLoadableElement {
     this.setAttribute('type', type);
   }
 
-  get text() {
```

Comment:
Ok, have we checked that this continues to behave per spec when you mutate the `<script/>` and such?

This seems like something that would break a _lot_ of things.